ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -2331,11 +2367,15 @@ router_bgp:
     neighbor_default:
       encapsulation: < vxlan | mpls >
       next_hop_self_source_interface: < source interface >
+      next_hop_self_received_evpn_routes:
+        enable: < true | false >
+        inter_domain: < true | false >
```

Comment:
We have to mention that this `inter_domain:` will narrow it down only to inter-domain routes, if it's false it will advertise both intra/inter. not sure if this is needed?

Reply:
Since it's a valid cli option i think we should keep it here. We can always decide to always set inter_domain: true in eos_designs later on.